src/Facade/Services/LinnDeliveryAddressService.cs

```diff
@@ -34,18 +31,11 @@ public LinnDeliveryAddressService(IRepository<LinnDeliveryAddress, int> reposito
             throw new NotImplementedException();
         }
 
-        protected override void DeleteOrObsoleteResource(
-            LinnDeliveryAddress entity,
-            IEnumerable<string> privileges = null)
-        {
+        protected override void DeleteOrObsoleteResource(LinnDeliveryAddress entity, IEnumerable<string> privileges = null)        {
             throw new NotImplementedException();
         }
 
-        protected override void UpdateFromResource(
-            LinnDeliveryAddress entity, 
-            LinnDeliveryAddressResource updateResource,
-            IEnumerable<string> privileges = null)
-        {
+        protected override void UpdateFromResource(LinnDeliveryAddress entity, LinnDeliveryAddressResource updateResource, IEnumerable<string> privileges = null)        {
```

Comment:
I prefer the parameters each on a new line when they get lengthy like this

Reply:
Aye, I'll do that for each of the Facade service methods. It is a little lengthy.

Reply:
the formatting is wrong too as per my last mini review. Is your style cop working?